### PR TITLE
Added TripMode Cask

### PR DESCRIPTION
**Pull request by winkelsdorf**

### CODE REVIEW

File: Casks/tripmode.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask :v1 => 'tripmode' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://tripmode.ch/TripMode.pkg'
+  name 'TripMode'
+  # AppCast not up-to-date yet
```
Comment (vitorgalvao) [inline Casks/tripmode.rb]: You can remove this comment and uncomment appcast. It’ll be a long while before we can take advantage of it anyway.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

File: Casks/tripmode.rb (hunk 1 above)
Comment (winkelsdorf) [inline Casks/tripmode.rb]: Ok, updating it

**Comment by winkelsdorf** [comment]
Sure, doing it now

**Comment by vitorgalvao** [comment]
Seems like you added a few more, instead.

**Comment by winkelsdorf** [comment]
Haha, I know. Thanks to SourceTree, but console fixed it ;)

**Comment by winkelsdorf** [comment]
Thanks for the merge :)

**Comment by vitorgalvao** [comment]
Thank you for the continued good contributions.

**Comment by winkelsdorf** [comment]
You're welcome :)

**Pull request merged**